macros/optimizers/index.sql

```diff
@@ -1,5 +1,5 @@
 {% macro index(this, column) %}
 
-create index if not exists "{{ this.name }}__index_on_{{ column }}" on {{ this }} ("{{ column }}")
```

Comment:
Is it common to prefix the index name with `schema_name.`?

Reply:
I.e. the index name should already be isolated to the schema, correct? If so, we probably don't need to include the schema in the index name.